code/game/machinery/anti_bluespace.dm

```diff
@@ -0,0 +1,115 @@
+/obj/machinery/anti_bluespace
+	name = "bluespace inhibitor"
+	desc = "Scrambles any bluespace related activity and displaces it away from the beacon's area of effect."
+	icon = 'icons/obj/telescience.dmi'
+	icon_state = "nopad"
+	anchored = 1
+	density = 1
+	use_power = 1
+	idle_power_usage = 500
```

Comment:
I would make it consume way more power

Reply:
I think the power level is fine however if there is a balancing issue with it, I will consider changes to power consumption.